docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
+
+  none  : Do not change boot device order
```

Comment:
I see, that makes more sense.

I still think we ought to make the API less IPMI-centric. In particular, having this set of options be defined here strikes me as a bad idea; they should be obm-specific. If we move this to a section of docs that specifically describes the IPMI backend, we can also just *say* "these have the same meaning as the options for ipmitool."